scripts/build/index.js

```diff
@@ -6,6 +6,7 @@ const {
   inputDir = join(process.cwd(), 'source'),
   outputDir = join(process.cwd(), 'dist'),
   basePath = '/',
+  assetPath = process.env.CDN_URL,
```

Comment:
If we call it asset URL also ... then there's less ambiguity. It's gotta be a full URL please.